_guides/znc.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: documentation
+title: Set up ZNC with The Lounge
+description: How to configure ZNC for use with The Lounge
+---
+
+## Why you may not need ZNC
+The Lounge already gives you most of the features you expect from a bouncer:
+
+- Scrollback is already available from within The Lounge, even across restarts, [if message storage is configured](/docs/configuration#messagestorage).
+
+- Multiple devices can connect to The Lounge at the same time, using The Lounge as a client.
+
+- The Lounge keeps a persistent connection to the IRC server even if you don't have the client open anywhere.
+
+- Push notifications are available if they are supported by your platform and browser.
+
+## Setting up ZNC
```

Comment:
```suggestion
## Set up ZNC
```

To be consistent with the next bullet (and most pages of the website, if I'm correct).